basics.js

```diff
@@ -31,6 +37,20 @@ def greet(name, excited)
 end
 */
 
+function greet(name, excited) {
+  greeting = "Hi, #{name} here!"
+  if (excited === true ) { greeting.upcase;
```

Comment:
No need to check that `excited === true` as it will automatically return a true/false value. But still correct. 

There is no .upcase method, and if there was you'd want to return the value of executing `.upcase()`.